### PR TITLE
Update User Guide - Clear and Exit

### LINKED ISSUE #54: Update User Guide: Clear all entries, Exiting the program

### PR DESCRIPTION
**Pull request by uyencfi**
Closes #54

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ It is optimized for **CLI users** so that frequent tasks can be done faster by t
 This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
 
 - Libraries used: [JavaFX](https://openjfx.io/), [Jackson](https://github.com/FasterXML/jackson), [JUnit5](https://github.com/junit-team/junit5)
+>>>>>>> upstream/master
```
Comment (chaiwanlin) [inline docs/index.md:23]:
```suggestion
```

**Review by chaiwanlin — CHANGES_REQUESTED** [review]
Just commit the change I made then LGTM

**Review by chaiwanlin — APPROVED** [review]
LGTM

**Review by caipng — APPROVED** [review]
LGTM!

**Pull request merged**